tests/apps/tethysapp-test_app/setup.py

```diff
@@ -23,12 +26,9 @@
     url='',
     license='',
     packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
+    package_data={'': resource_files},
```

Comment:
It seems weird that there is a dictionary with an empty key. Is this right?

Reply:
After reading some documentation (https://docs.python.org/3/distutils/setupscript.html and other websites) it seems like that is a convention that can be used. The other convention is {app_package: data_files} but if you look up where resource_files is created it already includes a reference to the app_package.   I think this is a correct way to do it.

Reply:
Resolved